DirectEmployers/MyJobs
# [PD-2833] Parse user agents to provide end-user with more readable information.

## What's this PR do?
Parses the user agent and adds specific parsed info to the AnalyticsEvent object.

## Where should the reviewer start?
- Made small modifications to the UserLocationBaseModel to be a bit more consistent
- Added the user agent parsing. Should ensure that user agent is always created before first save.

## How should this be manually tested?
Create objects every way you can with any user agent you can find. Make sure the info is set as you expected after the object is saved.
